CONTRIBUTING.md

```diff
@@ -2,25 +2,44 @@
 
 **This document applies during 2024 and may change at a future date.**
 
-### Introduction
+## Introduction
 
-The current status of the ASVS project is as follows:
+### What is [OWASP](https://owasp.org/)?
+
+The Open Worldwide Application Security Project (OWASP) is a nonprofit organization that works to improve the security of software. It has many programs to work towards this goal. One of those programs is the ASVS.
+
+### What is the [ASVS](https://owasp.org/www-project-application-security-verification-standard/)?
+
+The OWASP Application Security Verification Standard (ASVS) Project provides a basis for testing web application technical security controls and also provides developers with a list of requirements for secure development.
+
+The primary aim of the OWASP Application Security Verification Standard (ASVS) Project is to normalize the range in the coverage and level of rigor available in the market when it comes to performing Web application security verification using a commercially-workable open standard.
```

Comment:
This is a little wordy. Perhaps "The main goal of the OWASP Application Security Verification Standard (ASVS) Project is to make sure that Web application security checks are consistent and thorough, using an open standard that can be used in the market."

Reply:
How about:

> The OWASP Application Security Verification Standard (ASVS) Project provides a basis for testing web application technical security controls and also provides developers with a list of requirements for secure development.
>
> The project aims is to provide a commercially-usable open standard for web application security with a consistent level of coverage and rigor.

Reply:
```suggestion
The OWASP Application Security Verification Standard (ASVS) Project provides a basis for testing web application technical security controls and also provides developers with a list of requirements for secure development.

The project aims is to provide a commercially-usable open standard for web application security with a consistent level of coverage and rigor.
```